core/ui/UIMediaPlayer.cpp

```diff
@@ -136,20 +138,659 @@ struct PrivateVideoDescriptor
         PS_SET_UNIFORM_R(ps, "colorTransform", colorTransform);
     }
 };
+
+std::unique_ptr<MediaEngineFactory> _meFactory = MediaEngineFactory::create();
+
+const char* CIRCLE_IMAGE =
+    "iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/"
```

Comment:
convert to binary maybe better

Reply:
Oh! I actually missed that. We don't need it anymore, since I can just use the DrawNode code to generate it too.